File: Find the missing letter/Find the missing letter.js
const findMissingLetter = (array) => {
  for(let i=1;i<array.length;i++){
    if(array[i].charCodeAt(0)-array[i-1].charCodeAt(0)!==1){
      return String.fromCharCode(array[i].charCodeAt(0)-1)
    }
  }
}

File: Find the missing letter/Find the missing letter.swift
func findMissingLetter(_ chArr: [Character]) -> Character {
    for i in 1..<chArr.count {
        let prev = chArr[i - 1].asciiValue!
        let curr = chArr[i].asciiValue!
        if curr - prev != 1 {
            return Character(UnicodeScalar(curr - 1))
        }
    }
    return " "
}
